### PR TITLE
update user account and TFA emails and help for watching

### PR DESCRIPTION
**Pull request by lmirabile**
**Description:**
Updated emails as indicated in Jira ticket.

**Jira:**
https://jira.mass.gov/browse/DP-16624

**To Test:**
- [ ] Add steps to test this feature

**Screenshots/GIFs:**
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

---

[Peer Review Checklist](https://github.com/massgov/opemass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: docroot/modules/custom/mass_flagging/mass_flagging.module [hunk 1]
```diff
@@ -142,7 +142,7 @@ function mass_flagging_node_update_insert_send_notifications(Node $node) {
               '@node_edit_time' => $node_edit_time,
               ':diff_url' => $diff_url,
               '@author_message' => $author_message ? "\n\nRevision log message:\n\n" . $author_message : '',
-              ':help_url' => Url::fromRoute('help.page', ['name' => 'mass_flagging'])->setAbsolute()->toString(),
+//              ':help_url' => Url::fromRoute('help.page', ['name' => 'mass_flagging'])->setAbsolute()->toString(),
```
Comment (rbayliss) [inline docroot/modules/custom/mass_flagging/mass_flagging.module]: We should delete this line - it's causing test failures for code style.

**Review by rbayliss — CHANGES_REQUESTED** [review]
Looks good - just needs one small change.

File: docroot/modules/custom/mass_flagging/mass_flagging.module (hunk 1 above)
Comment (lmirabile) [reply docroot/modules/custom/mass_flagging/mass_flagging.module]: done

**Review by ygannett — APPROVED** [review]
LGTM

File: docroot/modules/custom/mass_flagging/mass_flagging.module (hunk 1 above)
Comment (ygannett) [reply docroot/modules/custom/mass_flagging/mass_flagging.module]: Resolve this as the requested change was done.

**Pull request merged**